chatgpt-on-your-data-2.mdx

```diff
@@ -0,0 +1,203 @@
+---
+title: 'Xata & OpenAI: ChatGPT for your data (Part 2)'
+description: 'Use Xata to ask your data questions and get the answers that matter most.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/chatgpt-on-your-data.png
```

Comment:
Used the same image from part 1, maybe we want to update it

Reply:
The updated image is here: https://github.com/xataio/mdx-blog/issues/83#issuecomment-1701345176

Reply:
Can you please update it here? 🙏